flocker/node/test/test_deploy.py

```diff
@@ -626,32 +697,50 @@ class ApplicationNodeDeployerDiscoverNodeConfigurationTests(
     Tests for ``ApplicationNodeDeployer.discover_local_state``.
     """
     def setUp(self):
+        self.hostname = u"example.com"
         self.network = make_memory_network()
         self.node_uuid = uuid4()
         # https://clusterhq.atlassian.net/browse/FLOC-1926
         self.EMPTY_NODESTATE = NodeState(
-            hostname=u"example.com",
+            hostname=self.hostname,
             uuid=self.node_uuid,
             manifestations={}, devices={}, paths={},
             applications=[])
 
-    def test_discover_none(self):
+    def _verify_discover_state_applications(
+            self, units, expected_applications,
+            current_state=None, start_applications=False):
         """
-        ``ApplicationNodeDeployer.discover_state`` returns an empty
-        ``NodeState`` if there are no Docker containers on the host.
+        Utility method for common code across the test cases in this class.
```

Comment:
The fact it's utility can be deduced from its name and location; the interesting thing is what it actually does.